census_view.hpp

```diff
@@ -134,6 +134,14 @@ class CensusView final
     wxDataViewCtrl* list_window_;
     wxObjectDataPtr<CensusViewDataViewModel> list_model_;
 
+    struct header_cache
```

Comment:
Sorry for nitpicking, but I don't think this is a good name because this is not really a cache at all. I'd call it just `header_column` because it stores information about a column.